### PR TITLE
<Tooltip /> - Style 

### PR DESCRIPTION
**Pull request by liorzisman**
Adding style and connecting props for the <Tooltip />

### CODE REVIEW

**Comment by lbelinsk** [comment]
Pr looks great!
Did 2 tiny commits, nothing worth mentioning.
Tagged you in an issue that should contain all Tooltip issues / future needed features

**Comment by liorzisman** [comment]
@lbelinsk added everything that was missing. please review

File: packages/wix-ui-core/src/components/Tooltip/Tooltip.driver.ts [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
-  mouseEnter: () => eventTrigger.mouseEnter(element)
-});
+export const tooltipDriverFactory = ({element, componentInstance, eventTrigger}) => {
+  let domTestkit = null;
+
+  if (componentInstance) {
+    domTestkit =  new DomTestkit({componentId: componentInstance.id});
+  }
+
+  return {
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Tooltip/Tooltip.driver.ts]: Missing comments for each method here. (Sorry :))

File: packages/wix-ui-core/src/components/Tooltip/styles.ts [hunk 2]
```diff
@@ -5,5 +5,17 @@ export const styles = (theme: TooltipTheme) => {
...
+
+      borderWidth: theme.borderWidth,
+      borderStyle: theme.borderStyle,
+      borderColor: theme.borderColor,
+      borderRadius: theme.borderRadius,
+
+      padding: theme.contentPadding
+    },
+
+    tooltipArrow: {}
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Tooltip/styles.ts]: I think you forgot this one, can be removed

File: packages/wix-ui-core/src/components/Tooltip/Tooltip.driver.ts [hunk 3]
```diff
@@ -1,9 +1,27 @@
+import {DomTestkit} from 'wix-ui-jss/domTestkit';
```
[the remark this answers is not in the document]
Comment (liorzisman) [reply packages/wix-ui-core/src/components/Tooltip/Tooltip.driver.ts]: epic :) glad you liked

File: packages/wix-ui-core/src/components/Tooltip/Tooltip.driver.ts (hunk 1 above)
Comment (liorzisman) [reply packages/wix-ui-core/src/components/Tooltip/Tooltip.driver.ts]: done

File: packages/wix-ui-core/src/components/Tooltip/styles.ts (hunk 2 above)
Comment (liorzisman) [reply packages/wix-ui-core/src/components/Tooltip/styles.ts]: whoops! removed

**Pull request merged**